daily-challenges/2024-09/16/thaoha93.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { log } from "console";
+
+function reverseString(originalString:string){
+    var newString = "";
+    for (let i = originalString.length - 1; i >= 0; i--) {
+        newString += originalString[i];
+    }
+    return newString;
+}
+console.log(reverseString('hello'));
+
+//Playwright
+import { expect, test } from '@playwright/test';
+const testData = {
+    userName: "thaoha",
+    email: "lily@gmail.com",
+    interests: ['Technology','Science','Music'],
+    country: "Canada",
+    dateOfBirth: "1993-09-29",
+    profilePicture: "daily-challenges/2024-09/16/00-problem.md",
+    biography: "Think outsite of the box",
+    rateUs: "9",
+    favColor: "#00ffaa"
+  }
+
+test('Day 16', async ({ page }) => {
+    await page.goto('https://material.playwrightvn.com/');
+
+    await page.locator('[href|="01"]').click();
+
+    const textName = await page.locator('#username');
+    await textName.clear();
+    await textName.fill(testData.userName);
+
+    const txtemail = await page.locator('#email');
+    await txtemail.clear();
+    await txtemail.fill(testData.email);
+
+    await page.locator('#female').click();
+
```

Comment:
Not a big problem, nhưng những dòng action có thể để liền nhau (giống phần expect để trông đồng bộ hơn.